cognite/client/_api/time_series.py

```diff
@@ -612,12 +637,19 @@ def update(
                 >>> my_update = TimeSeriesUpdate(id=1).description.set("New description").metadata.add({"key": "value"})
                 >>> res = client.time_series.update(my_update)
         """
+        headers: dict | None = None
+        if (isinstance(item, Sequence) and any(ts.instance_id for ts in item)) or (
+            isinstance(item, (TimeSeries, TimeSeriesWrite, TimeSeriesUpdate)) and item.instance_id
+        ):
```

Comment:
I don't see you do
`isinstance(item, (TimeSeries, TimeSeriesWrite, TimeSeriesUpdate))`
for the elements in the sequence - does this mean we can remove the check, or should it be added both places?

Reply:
Good catch the instance check was added before Instance ID was supported in TimeSeriesUpdate.